client/my-sites/site-settings/media-settings.jsx

```diff
@@ -98,6 +100,23 @@ const MediaSettings = ( {
 					</FormSelect>
 				</div>
 			</FormFieldset>
+
+			<FormFieldset className="media-settings__formfieldset has-divider is-top-only">
+				<div className="media-settings__info-link-container site-settings__info-link-container">
+					<InfoPopover position="left">
+						<ExternalLink target="_blank" icon href="https://jetpack.com/support/videopress/" >
+							{ translate( 'Learn more about VideoPress.' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **33** times:
`translate( 'Learn more about CSS' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).